fix(CommandHandler): deal with path resolution agnostically

Codeize opened the pull request:
fixes a traceback error that arose when trying to deploy commands on a windows based machine.

Codeize committed:
fix(CommandHandler#registerCommands): deal with path resolution agnostically

fixes a traceback error that arose when trying to deploy commands on a windows based machine.

didinele reviewed (changes requested):
lgtm, just also deal with it for components (there are none right now, hence the lack of errors, but if we add any we'll run into the same problem)

https://github.com/ChatSift/ModMail/blob/09ab5cf7351652df3bc2afcb08c77b1f42f009df/packages/bot/src/struct/CommandHandler.ts#L186

Codeize committed:
fix(CommandHandler#registerComponents): deal with path resolution agnostically

will fix a traceback error that will arise when trying to deploy components on a windows based machine.
